third-party/llvm/Makefile

```diff
@@ -45,6 +45,8 @@ endif
 
 CMAKE ?= cmake
 
+CHPL_LLVM_SUPPORTED_VERSIONS := $(shell python -c 'import sys; sys.path.insert(0, "${CHPL_MAKE_HOME}"+"/util/chplenv"); import chpl_llvm; print(chpl_llvm.llvm_versions_string())')
```

Comment:
I don't think this will work on systems that dropped the `python` alias. See `util/printchplenv` or `util/config/compileline` for an example of how we select the python name portability for core scripts.

I would suggest creating a `util/config/llvmVersions` or something that does something similar. Might also be a little cleaner to add a `--suported-versions` option to `chpl_llvm` to avoid having to call an internal function.

Reply:
OK, thanks for clarifying. I think I handled your suggestion right this time.